packages/xo-server-backup-reports/src/index.js

```diff
@@ -152,7 +152,11 @@ class BackupReportsXoPlugin {
 
   async _backupNgListener (_1, _2, schedule, runJobId) {
     const xo = this._xo
-    const log = await xo.getBackupNgLogs(runJobId)
+    const log = await xo.getBackupNgLogs(runJobId.trim())
+
+    if (log === undefined) {
+      throw new Error(`the log with the runId (${runJobId}) is not found`)
```

Comment:
Use the used runId (trimmed), otherwise it might be confusing :slightly_smiling_face: